Lecture1/Task1/task1/task1.java

```diff
@@ -0,0 +1,26 @@
+package task1;
+        import java.util.Scanner;
+        import java.util.Arrays;
+
+public class task1 {
```

Comment:
Please check out the Java naming convention ([link](https://www.oracle.com/technetwork/java/codeconventions-135099.html))